### DISCUSSION TITLE
Network Adapters / IPs Are Missing from Scrypted UI

### DISCUSSION THREAD

**Discussion by timothystewart6** [koush/scrypted#1436] [Q&A] [answered]
Hi.  I am not certain if this is a bug or a feature so I opened a discussion.  It seems I cannot select my network adapter for the the server address because it's missing.

e.g.

Inside container:

```console
# ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: net1@if4: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default 
    link/ether c6:5e:c4:8a:7a:69 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 192.168.40.201/24 brd 192.168.40.255 scope global net1
       valid_lft forever preferred_lft forever
    inet6 fe80::c45e:c4ff:fe8a:7a69/64 scope link 
       valid_lft forever preferred_lft forever
56: eth0@if57: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 22:cf:5f:6d:04:ea brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 10.42.4.190/32 scope global eth0
       valid_lft forever preferred_lft forever
    inet6 fe80::20cf:5fff:fe6d:4ea/64 scope link 
       valid_lft forever preferred_lft forever
```

In Scrypted UI:

![image](https://github.com/koush/scrypted/assets/1322205/c11bbb2d-eefc-4222-ba06-7744529b0ad4)

I would like to be able to choose `net1` to help troubleshoot a HomeKit container issue and noticed that it's missing some of my adapters and IPs.   If I can't set the Server Address, I can't then manually set it for the HomeKit plugin.  That aside, shouldn't it show up here? Is there a filter on this?

**Comment by timothystewart6** [comment]
I was able to get it added to HomeKit, but still curious why these are filtered in the UI.

**Comment by timothystewart6** [comment]
Ended up opening a PR to fix this
https://github.com/koush/scrypted/pull/1437

**Comment by koush** [comment, marked as answer]
intention is to filters out potentially problematic addresses. fwiw, you can type whatever address you want in that box manually.
